api.md

```diff
@@ -1763,6 +1759,10 @@ WITH (timescaledb.continuous) AS
     GROUP BY time_bucket('1h', timec);
 ```
 
+>:TIP: In order to keep the continuous aggregate up to date with incoming data,
```

Comment:
Doh! Great question.

In theory, since none of these examples show turning it off, there's no reason to actually say this now, right?

Reply:
I agree with @mfreed that we don't need to talk about this any more. If user specified `end_offset = 0`, it will refreshing cagg using the latest data.